tests/runner/src/omvf.rs

```diff
@@ -0,0 +1,261 @@
+use anyhow::{bail, Result};
+use sha2::{Digest, Sha256};
+use std::{
+    env,
+    io::{Cursor, Read},
+    path::{Path, PathBuf},
+};
+use tar::Archive;
+use tempfile::TempDir;
+use ureq::Agent;
+#[cfg(target_os = "linux")]
+use {std::fs::Permissions, std::os::unix::fs::PermissionsExt};
+
+/// Name of the ovmf-prebuilt release tag.
+const OVMF_PREBUILT_TAG: &str = "edk2-stable202311-r2";
```

Comment:
This is a pretty outdated version, right?

Reply:
Yes. It's about a year old. I chose this version as it is the one the uefi crate uses in their tests. 
We can probably update this. It just wasn't a priority.

Reply:
I just tested the latest tarball version(`edk2-stable202405-r1`) released to https://github.com/rust-osdev/ovmf-prebuilt.

It breaks the uefi pxe tests.  I don't really know enough about pxe to debug this, but it looks like it does not find the bootloader and tries to start a uefi terminal.